### PR TITLE
Fix link to https://www.kitura.dev

### PR DESCRIPTION
**Pull request by kikeenrique**
Fix link that was not working

### CODE REVIEW

**Review by dannys42 — APPROVED** [review]

**Comment by kikeenrique** [comment]
As this change is trivial and unlikely to break CI build. Would it be ready to merge?

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Kitura is a web framework and web server that is created for web services writte
 
 ## Getting Started
 
-Visit [www.kitura.io](http://www.kitura.io/) for a Getting Started guide, tutorials, and API reference documentation.
+Visit [https://www.kitura.dev](http://www.kitura.dev/) for a Getting Started guide, tutorials, and API reference documentation.
```
Comment (dannys42) [inline README.md:53]: Would you mind setting renaming http to https in the text as well?

**Comment by dannys42** [comment]
@kikeenrique Yeah, we'll figure out the CI separately.  Thanks for the update.

File: README.md (hunk 1 above)
Comment (kikeenrique) [reply README.md]: Done. And I've modified 2 more links.
There is also a link pointing to https://www.kitura.io/packages.html#al, but I don't know where it should point now.

**Review by dannys42 — APPROVED** [review]
Thanks!

**Pull request merged**